plugins/modules/network_settings_workflow_manager.py

```diff
@@ -347,15 +347,42 @@
                     description: Retain existing banner message.
                     type: bool
                 type: dict
+              wired_data_collection:
+                description: Enables or disables the collection of data from wired network devices for telemetry and monitoring purposes.
+                    Applicable from sdk version 2.3.7.6 onwards.
+                suboptions:
+                    enable_wired_data_collection:
+                      description: Enable or disable wired data collection.
+                      type: bool
+                      default: false
+                type: dict
+              wireless_telemetry:
+                description: Enables or disables the collection of telemetry data from wireless network devices for performance monitoring and analysis.
+                    Applicable from sdk version 2.3.7.6 onwards.
+                suboptions:
+                    enable_wired_data_telemetry:
+                      description: Enable or disable wireless telemetry.
+                      type: bool
+                      default: false
+                type: dict
               netflow_collector:
                 description: Netflow collector details under a specific site.
                 suboptions:
+                  collector_type:
+                    description: Type of NetFlow collector (eg Telemetry_broker_or_UDP_director). Applicable from sdk version 2.3.7.6 onwards.
```

Comment:
description: Type of NetFlow collector. Supported values include 'Builtin' and 'Telemetry_broker_or_UDP_director'. Applicable from Cisco Catalyst Center version 2.3.7.6 onwards.

Reply:
Changed.